gcn_classic_to_kafka/heartbeat.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+import asyncio
+import datetime
+import json
+import logging
+
+import confluent_kafka
+
+log = logging.getLogger(__name__)
+
+TIMEOUT = 1
+TOPIC = "gcn.heartbeat"
+
+
+async def run(producer: confluent_kafka.Producer):
```

Comment:
Does this work? I thought the producer would need more configuration options, like the bootstrap servers

Reply:
The producer is created in cli.py. The connection object is passed to this function. It's the same connection object that is passed to the function in socket.py.

Reply:
Ahh okay nice, thanks